### PR TITLE
Fixed call on undefined object

### PR DESCRIPTION
**Pull request by wscheffer**

### CODE REVIEW

File: www/app/page5.controller.js [hunk 1]
```diff
@@ -12,18 +12,18 @@ module.exports = function ($scope) {
...
 
+  $scope.currentroom = document.domain;
   $scope.nick = "Anon";
-  //$scope.currentroom = document.domain;
   $scope.rooms = [];
   $scope.messages = [];
   $scope.newMessage = {};
 
   //For localhost testing
   if (document.domain == 'localhost') {
```
Comment (jaderabbit) [inline www/app/page5.controller.js:22]: Why is this needed? this will mean that anything NOT at local host WON'T get to:

```
     $scope.rooms.push({
        url: $scope.currentroom,
        key: $scope.currentroom.hashCode()
      });
```

**Review by jaderabbit — CHANGES_REQUESTED** [review]

File: www/app/page5.controller.js (hunk 1 above)
Comment (wscheffer) [reply www/app/page5.controller.js:22]: I think the code should actually be 

```
 //For localhost testing to add the port
 if (document.domain == 'localhost') {
    $scope.currentroom = $scope.currentroom + ':8080';
  }

//Push the current domain to $scope.rooms so that the ng-model on ng-select works $scope.rooms.push({
      url: $scope.currentroom,
      key: $scope.currentroom.hashCode()
});
```

Comment (wscheffer) [reply www/app/page5.controller.js:22]: I cant change it to this and publish and see if it works?

**Comment by jaderabbit** [comment]
🍭

**Review by jaderabbit — APPROVED** [review]
